azure-pipelines.yml

```diff
@@ -65,6 +65,32 @@ jobs:
             displayName: "npm build and test"
           - script: npm run test:nolint
             displayName: "npm build and test (no linting)"
+    - job: TestExportedTypes
+      pool:
+          name: "1ES-Hosted-AzFunc"
+          demands:
+              - ImageOverride -equals MMSUbuntu20.04TLS
+      steps:
+          - task: NodeTool@0
+            inputs:
+                versionSpec: $(NODE_14)
+            displayName: "Install Node.js"
+          - script: npm install
+            displayName: "Install Node.js Dependencies"
+          - script: npm run build
+            displayName: "Build project"
+          - script: npm pack
+            displayName: "Pack npm package"
+          - task: Bash@3
+            displayName: "Create test app"
+            inputs:
+                targetType: "inline"
+                script: "mkdir $(System.DefaultWorkingDirectory)/../test-app && cp -r $(System.DefaultWorkingDirectory)/samples/UnitTesting $(System.DefaultWorkingDirectory)/../test-app"
+          - task: Bash@3
+            displayName: "Build test app"
+            inputs:
+                targetType: "inline"
+                script: "cd $(System.DefaultWorkingDirectory)/../test-app/UnitTesting && npm install && npm uninstall durable-functions && pckg=$(ls $(System.DefaultWorkingDirectory)/*.tgz) && npm install $pckg && npm run build"

```

Comment:
Should we make the value of `pckg` more concrete? So far it just grabs any `.tgz` file. Maybe we can give it the durable functions package prefix or some known substring that is a part of the generated `tgz` name to ensure we're grabbing the right payload

Reply:
Makes sense, I added the `durable-functions-` prefix